clusters/build-clusters/01_cluster/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -82,3 +82,31 @@ items:
     - get
     - list
     - watch
+
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: ci-admins-secret
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - secrets
+      verbs:
+        - get
```

Comment:
this should be part of the rule above this, no?

Reply:
Removed.